node-export-server/server.js

```diff
@@ -31,7 +31,8 @@ app.get('/options', function(req, res) {
     res.json({
         Datasource: _.keys(config.datasources),
         Schema: _.keys(config.schemas),
-        Format: _.keys(config.formats)
+        Format: _.keys(config.formats),
+        SetTagOverrides: JSON.stringify(config.tagOverrides)
```

Comment:
Putting this in the options result has this undesired effect without further UI changes.  (Defaults to a drop down, but doesn't know how to handle the stringified json.  Other options just present an array used in the dropdown).
![screenshot from 2018-01-08 11-52-18](https://user-images.githubusercontent.com/2600932/34682233-4361e832-f46c-11e7-9bc5-ae562192bb1c.png)